apps/web/components/settings/platform/hooks/usePlatformMe.ts

```diff
@@ -6,14 +6,26 @@ export const usePlatformMe = () => {
   const QUERY_KEY = "get-platform-me";
   const platformMeQuery = useQuery<UserResponse>({
     queryKey: [QUERY_KEY],
-    queryFn: async (): Promise<UserResponse> => {
+    retryOnMount: false,
+    staleTime: 300000,
+    retry: (failureCount, error) => {
+      if (error.message === "Internal Server Error") {
```

Comment:
Better to use a code for this or rely on statusCode being 5xx. Text might start getting translated, causing weird issues.